modules/combined/test/tests/internal_volume/tests

```diff
@@ -111,50 +127,58 @@
     cli_args = 'Postprocessors/internalVolume/component=1
                 Outputs/file_base=rz2_out'
     prereq = rz
+    requirement = 'The system shall compute internal volume given an enclosing sideset for axisymmetric coordinates and with the y coordinate'
```

Comment:
Similar to the solid mechanics version of the test.

Reply:
Done.